config/v1/types_infrastructure.go

```diff
@@ -573,6 +583,42 @@ type EquinixMetalPlatformStatus struct {
 	IngressIP string `json:"ingressIP,omitempty"`
 }
 
+// PowervsServiceEndpoint store the configuration of a custom url to
+// override existing defaults of PowerVS Services.
+type PowerVSServiceEndpoint struct {
+	// Name is the name of the Power VS services.
+	// Note that not all locations incude Power VS.
+	//
+	// +kubebuilder:validation:Pattern=`^[a-z0-9-]+$`
+	Name string `json:"name"`
+
+	// url is fully qualified URI with scheme https, that overrides the default generated
+	// endpoint for a client.
+	// This must be provided and cannot be empty.
+	//
+	// +kubebuilder:validation:Pattern=`^https://`
+	URL string `json:"url"`
+}
+
+// PowerVSPlatformSpec holds the desired state of the Power Systems Virtual Servers infrastructure provider.
+// This only includes fields that can be modified in the cluster.
+type PowerVSPlatformSpec struct{}
+
+// PowerVSPlatformStatus holds the current status of the Power VS infrastructure provider.
+type PowerVSPlatformStatus struct {
+	// Region holds the default Power VS region for new Power VS resources created by the cluster.
+	Region string `json:"region"`
+
+	// Zone holds the default colo zone for the new Power VS resources created by the cluster.
+	// Note: Currently only single-zone OCP clusters are supported
```

Comment:
Is "colo zone" established terminology, or could we spell out "collocation zone" (assuming that's what "colo zone" is short for)?
```suggestion
	// zone holds the default collocation zone for the new Power VS resources created by the cluster.
	// Note: Currently only single-zone OCP clusters are supported.
```